### PR TITLE
WEB-33694 - New resource for top options

### PR DESCRIPTION
**Pull request by bjoernmayer**
New endpoint to retrieve happy little top options

### CODE REVIEW

File: src/Response/TopOptions/Tag.php [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    }
+
+    /**
+     * @return string
+     */
+    public function __toString()
+    {
+        return (string) $this->id;
+    }
+
```
Comment (Exeu) [inline src/Response/TopOptions/Tag.php]: Drop blank line

File: src/Response/TopOptions/TopOption.php [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+        foreach ($rateAttributes as $rateAttribute) {
+            $this->rateAttributes[] = new RateAttribute($rateAttribute['value']);
+        }
+    }
+
+    /**
+     * @return int
+     */
+    public function getId() {
```
Comment (Exeu) [inline src/Response/TopOptions/TopOption.php]: inconsistent style guide

File: tests/_fixtures/top_options_response_200.php [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
...AFRyaXZhZ29cVGFzXFJlc3BvbnNlXFJlc3BvbnNlAGhlYWRlcnMiO2E6MTU6e3M6NDoiZGF0ZSI7czoyOToiTW9uLCAwNiBGZWIgMjAxNyAxNTowNzowNSBHTVQiO3M6Njoic2VydmVyIjtzOjQ3OiJBcGFjaGUvMi40LjEwIChEZWJpYW4pIFBIUC83LjAuNiBPcGVuU1NMLzEuMC4xayI7czoxMjoieC1wb3dlcmVkLWJ5IjtzOjk6IlBIUC83LjAuNiI7czoxMDoic2V0LWNvb2tpZSI7YToxOntpOjA7czo4MjoiWERFQlVHX1NFU1NJT049WERFQlVHOyBleHBpcmVzPU1vbiwgMDYtRmViLTIwMTcgMTY6MDc6MDUgR01UOyBNYXgtQWdlPTM2MDA7IHBhdGg9LyI7fXM6MTM6ImNhY2hlLWNvbnRyb2wiO3M6ODoibm8tY2FjaGUiO3M6MTM6IngtYXBpLXZlcnNpb24iO3M6MTA6InYzMV8wMV82YWEiO3M6Mjg6ImFjY2Vzcy1jb250cm9sLWFsbG93LWhlYWRlcnMiO3M6MTE1OiJBY2NlcHQsQWNjZXB0LUxhbmd1YWdlLFVzZXItQWdlbnQsQXV0aG9yaXphdGlvbixYLVRydi1BdXRob3JpemF0aW9uLENhY2hlLUNvbnRyb2wsQ29udGVudC1UeXBlLENvb2tpZSxYLVRydi1BcGktS2V5IjtzOjI4OiJhY2Nlc3MtY29udHJvbC1hbGxvdy1tZXRob2RzIjtzOjI3OiJPUFRJT05TLFBPU1QsR0VULFBVVCxERUxFVEUiO3M6Mjc6ImFjY2Vzcy1jb250cm9sLWFsbG93LW9yaWdpbiI7czoxOiIqIjtzOjMyOiJhY2Nlc3MtY29udHJvbC1hbGxvdy1jcmVkZW50aWFscyI7czo0OiJ0cnVlIjtzOjEzOiJhY2NlcHQtcmFuZ2VzIjtzOjQ6Im5vbmUiO3M6NDoidmFyeSI7czoxNToiQWNjZXB0LUVuY29kaW5nIjtzOjE2OiJjb250ZW50LWVuY29kaW5nIjtzOjQ6Imd6aXAiO3M6MTQ6ImNvbnRlbnQtbGVuZ3RoIjtzOjM6IjI3NSI7czoxMjoiY29udGVudC10eXBlIjtzOjE2OiJhcHBsaWNhdGlvbi9qc29uIjt9fQ=='));
```
Comment (Exeu) [inline tests/_fixtures/top_options_response_200.php]: add a new line in the end of the file

File: docs/reference/trivago/tas/tas.rst [hunk 4]
```diff
@@ -300,3 +300,39 @@ Response
...
+    foreach ($topOptions as $topOption) {
+        // ...
+    }
+
+Request
+-------
+
+The request must be an instance of :code:`Trivago\Tas\Request\TopOptionsRequest`. It has no parameters.
+
++------------+-----------+---------+------------+-----------------------------------------------------------+
```
Comment (Exeu) [inline docs/reference/trivago/tas/tas.rst]: Since we dont have any parameters we should drop this table

Comment (bjoernmayer) [inline docs/reference/trivago/tas/tas.rst]: NO

**Pull request merged**